colossalai/zero/sharded_model/sharded_model_v2.py

```diff
@@ -134,10 +134,14 @@ def forward(self, *args: Any, **kwargs: Any) -> torch.Tensor:
     def backward(self, loss):
         loss.backward()
         self._post_backward_operations()
+        for ophook in self._ophook_list:
+            ophook.post_iter()
 
     def backward_by_grad(self, tensor, grad):
         torch.autograd.backward(tensors=tensor, grad_tensors=grad)
         self._post_backward_operations()
+        for ophook in self._ophook_list:
+            ophook.post_iter()
 
```

Comment:
why not move then to _post_backward_operations?

Reply:
I think `_post_backward_operations()` just contains `ZeRO`'s own logic, so I just leave this code snippet here.

Reply:
This code snippet may be more general than `_post_backward_operations()`, I think.

Reply:
OK